rrmngmnt/ssh.py

```diff
@@ -203,6 +210,24 @@ def run(self, input_, timeout=None, get_pty=False):
                 self.err = normalize_string(err.read())
             return self.rc, self.out, self.err
 
+        def run_real_time(self, timeout=None, get_pty=False):
```

Comment:
I would change it to `run_interactive` or something like that?

In addition I would make it generator, and which returns a line (or tuple actually stdout, stderr) on every __next__ call.

In this way a caller can decide what to do with that lines ... so you have method which has more use-cases. an user can decide if these lines should be logged or processed further.

Then you don't need to `playbook_logger=None` parameter to generic `Host` class.
In general whole this change will be more lightweight in the matter of Host class.